docs/unicode.md

```diff
@@ -0,0 +1,68 @@
+# Macros And Unicode
+Macros are used for sending multiple keystrokes in a single action. This is useful for
+things like unicode input, sending strings of text, or other automation.
+
+## Basic Macros
+The most basic macro is send_string(). It can be used to send any typed keycode, including the return and tab key.
+```python
+from kmk.macros.simple import send_string
+
+WOW = send_string("Wow, KMK is awesome!")
+
+keymap = [...WOW,...]
+```
+# Unicode
+Before using unicode mode, you will need to set your platform. This can be done either of these ways.
+
+	unicode_mode = UnicodeModes.LINUX
+	OR
+	keymap = [...KC.UC_MODE_LINUX,...]
```

Comment:
If they do this, they'll need to tap that keycode before unicode mode will work at all, this is absolutely worth noting.

It's also worth noting that using that key, they can change at runtime, allowing them to flip from a Linux to MacOS machine with no keyboard flashing, for example.